src/test/java/net/smartcosmos/dto/metadata/MetadataCreateTest.java

```diff
@@ -0,0 +1,54 @@
+package net.smartcosmos.dto.metadata;
```

Comment:
We don't want to test every getter and setter explicitly since that is just busy work.  I found it interesting that the tests did not have 100% coverage and wanted to point that out.  I should have been more clear.